src/cwv/handler.js

```diff
@@ -44,7 +45,69 @@ export async function CWVRunner(auditUrl, context, site) {
   };
 }
 
+export async function convertToOppty(auditUrl, auditData, context) {
+  const {
+    dataAccess,
+    log,
+  } = context;
+
+  const opportunities = await dataAccess.Opportunity.allBySiteIdAndStatus(auditData.siteId, 'NEW');
+  let opportunity = opportunities.find((oppty) => oppty.getType() === AUDIT_TYPE);
+
+  if (!opportunity) {
+    const opportunityData = {
+      siteId: auditData.siteId,
+      auditId: auditData.id,
+      runbook: 'https://adobe.sharepoint.com/sites/aemsites-engineering/Shared%20Documents/3%20-%20Experience%20Success/SpaceCat/Runbooks/Experience_Success_Studio_CWV_CLS_Runbook.docx?web=1',
+      type: AUDIT_TYPE,
+      origin: 'AUTOMATION',
+      title: 'Core Web Vitals',
+      description: 'Core Web Vitals are key metrics Google uses to evaluate website performance, impacting SEO rankings by measuring user experience.',
+      guidance: {
+        steps: [
+          'Analyze CWV data using RUM and PageSpeed Insights to identify performance bottlenecks.',
+          'Optimize CWV metrics (CLS, INP, LCP) by addressing common issues such as slow server response times, unoptimized assets, excessive JavaScript, and layout instability.',
+          'Test the implemented changes with tools like Chrome DevTools or PageSpeed Insights to verify improvements.',
+          'Monitor performance over time to ensure consistent CWV scores across devices.',
+        ],
+      },
+      tags: [
+        'SEO',
```

Comment:
Can you update the tags as per mocks to just these 2 below:
* Traffic acquisition
* Engagement

Reply:
done ✅